event/serializers.py

```diff
@@ -45,14 +45,14 @@ class EventSerializer(serializers.ModelSerializer):
     organisation = OrganisationSerializer()
     location = EventLocationSerializer()
     sponser = EventSponserSerializer(many=True)
-    comments = EventCommentSerializer(many=True)
+    comment = EventCommentSerializer(many=True)
```

Comment:
@ZenabKhan this should be comments no?

Reply:
@mashhoodr all other fields are with a signular name. 
So just made it consistent. Should I revert back?

Reply:
Actually fix all of htem, if a variable has multiple (array) then it should be plural

Reply:
Fixed.